spec/services/jwt/generate_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Jwt::Generate do
+  context '.initialize' do
+    #mostrar que ele sempre gera um jti diferente
+    subject { described_class.new(user_id: 1) }
+
+    it 'has correct payload' do
+      expect(subject.payload).to include(:exp, :sub, :env, :jti)
+    end
+
+    it 'aways returns different tokens' do
+      expect(subject.token).not_to eq(described_class.new(user_id: 1).token)
+    end
+
+    it 'returns correct parse to string' do
+      expect(subject.to_s).to eq(subject.token)
+    end
+
+    it 'returns correct parse to hash' do
+      expect(subject.to_h).to eq({ access_token: subject.token })
+    end
+
+    it 'returns correct parse to json' do
+      expect(subject.to_json).to eq({ access_token: subject.token }.to_json)
+    end
+  end
+
+  describe '.token' do
+    subject { class_instance.token }
+    let(:class_instance) { described_class.new(user_id: 1) }
```

Comment:
Had to save class instance since SecureRandom is not mocked

Reply:
this should be the subject